napari/_vispy/camera.py

```diff
@@ -23,15 +23,18 @@ def __init__(self, view, camera, dims) -> None:
         self._dims = dims
 
         # Create 2D camera
-        self._2D_camera = PanZoomCamera(aspect=1)
+        self._2D_camera = add_mouse_pan_zoom_toggles(PanZoomCamera)(aspect=1)
         # flip y-axis to have correct alignment
         self._2D_camera.flip = (0, 1, 0)
         self._2D_camera.viewbox_key_event = viewbox_key_event
 
         # Create 3D camera
-        self._3D_camera = ArcballCamera(fov=0)
+        self._3D_camera = add_mouse_pan_zoom_toggles(ArcballCamera)(fov=0)
         self._3D_camera.viewbox_key_event = viewbox_key_event
 
+        # Set 2D camera by default
+        self._view.camera = self._2D_camera
```

Comment:
Moreover, there was a bug in this place. Without this line, the viewbox used `BaseCamera` by default, not `_2D_camera` instantiated above. But when you switched to the 3D mode, it used `_3D_camera`, and after switching to 2D again it eventually started using the `_2D_camera` created in the constructor.